qt/python/mantidqt/widgets/sliceviewer/view.py

```diff
@@ -566,5 +589,6 @@ def set_peaks_viewer_visible(self, on):
 
     # event handlers
     def closeEvent(self, event):
+        self._data_view.colorbar.closeEvent(event)
```

Comment:
Do you know if this is required now or is it a hangover from the original implementation?

Reply:
Ah yes just left over from before - I've removed it